modules/cas_cache/threads.c

```diff
@@ -277,8 +277,8 @@ int cas_create_metadata_updater_thread(ocf_metadata_updater_t mu)
 	int result;
 
 	result = _cas_create_thread(&info, _cas_metadata_updater_thread,
-			mu, CAS_CPUS_ALL, "ocf_metadata_updater_%d",
-			ocf_cache_get_id(ocf_metadata_updater_get_cache(mu)));
+			mu, CAS_CPUS_ALL, "ocf_metadata_updater_%s",
+			ocf_cache_get_name(ocf_metadata_updater_get_cache(mu)));
```

Comment:
"get_cache_id_string" instead of "ocf_cache_get_name" would be better - more compact and equivalent to the old version.

Reply:
I'd prefer to use full cache/core names in logged messages. When in some point in future we decide to let user specify names instead of using ids, messages will not need to be changed.